doc/api/sqlite.md

```diff
@@ -523,6 +523,27 @@ exception.
 | `TEXT`    | {string}             |
 | `BLOB`    | {Uint8Array}         |
 
+## `sqlite.backup(sourceDb, destination[, options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `sourceDb` {DatabaseSync} The database to backup. The source database must be open.
+* `destination` {string} The path where the backup will be created. If the file already exists, the contents will be
```

Comment:
If this is assumed to be a filesystem path, then handling it the same as our other fs APIs would be ideal. Paths can be expressed as either strings, `Buffer`, or `file://` scheme URLs. If you're not familiar, there are internal utilities for normalizing these into a usable path.

Reply:
Makes sense. I will search for examples to see how to do that.

Reply:
Maybe that? https://github.com/nodejs/node/blob/main/lib/internal/fs/utils.js#L700-L720

Reply:
This makes sense but I'd like to see this in another PR. DB locations are all defined as `strings` in other parts of sqlite module - e.g. [location in DatabaseSync](file:///Users/edysilva/projects/contributions/node/out/doc/api/sqlite.html#new-databasesynclocation-options).

We can have a PR dedicated to allow`Buffer` and URLs all at once.

Do you think it makes sense?

Reply:
I think that makes sense.